tests/modules/compile.bat

```diff
@@ -0,0 +1,5 @@
+del *.dxil
+del *.slang-module
+slangc.exe hit.slang -stage closesthit -entry closesthit -target dxil -o monolithic.dxil
```

Comment:
You can simply use multiple `//TEST:SIMPLE` lines in the .slang file to trigger the compile.